### ISSUE TITLE
Training on 8x 2080 Ti's

### ISSUE THREAD

**Issue by Snow-Sven** [mit-han-lab/bevfusion#374]
The code released is training on 8x A6000 GPUs. I only have access to 8x 2080 Ti's and I have not yet been able to successfully train the model. Will BEVFusion be able to train on 8x 2080 Ti's? If so, what extra steps need to be taken to achieve this and can you estimate how long it would take to train given that on the 8xA6000 the training took roughly 10 hours? If not, do you have any recommendations for solving or circumventing this issue?

**Comment by kentang-mit** [comment]
I think it might be difficult to train on 8X2080Tis due to the limited memory. I'm not sure whether it can hold batch=1 for the fusion model using the current implementation. 

But we are working on a new release of TorchSparse which could be viewed as a drop-in replacement of SpConv in this codebase. It is much faster and requires smaller memory. In this case there might be a possibility for you to run the model using batch=1 on 2080Ti. And you probably need gradient accumulation to partly recover the batch=4 performance.
